test/demo/ios-62-63/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="/build/payment-shim.js"></script>
+<script type="text/javascript">
+  var supportedInstruments = [{
+        supportedMethods: [
+            'basic-card',
+        ]
+    }];
+
+  var details = {
+      total: {
+          label: 'Total',
+          amount: {
+              currency: 'USD',
+              value: 55
+          }
+      },
+      displayItems: [{
+          label: 'Total amount',
+          amount: {
+              currency: 'USD',
+              value: 65
+          }
+      }, {
+          label: 'Discount',
+          amount: {
+              currency: 'USD',
+              value: -10
+          }
+      }, {
+          label: 'Shipping',
+          amount: {
+              currency: 'USD',
+              value: 0
+          }
+      }],
+      shippingOptions: [{
+          id: 'standard',
+          label: 'Standard shipping',
+          amount: {
+              currency: 'USD',
+              value: 0.00
+          },
+          selected: true
+      }, {
+          id: 'expedited',
+          label: 'Expedited shipping',
+          amount: {
+              currency: 'USD',
+              value: 0.9999
+          },
+          selected: false
+      }],
+      modifiers: [{
+          supportedMethods: ['basic-card'],
+          total: {
+              label: 'Total',
+              amount: {
+                  currency: 'USD',
+                  value: 45.00
+              }
+          },
+          additionalDisplayItems: [{
+              label: 'Visa discount',
+              amount: {
+                  currency: 'USD',
+                  value: -10.00
+              }
+          }],
+          data: {
+              supportedNetworks: ['visa'],
+          }
+      }]
+  };
+
+  var options = {
+      requestShipping: true
+  };
+
+  // Clone details to use as the updateWith() method argument.
+  let addressUpdateDetails = JSON.parse(JSON.stringify(details));
+  let optionUpdateDetails = JSON.parse(JSON.stringify(details));
```

Comment:
Let's be consistent with use of `let` vs `var` in this file. If this file undergoes the `let`->`var` conversion, let's use `let` and `const` for improved readability. If this file does not get converted, please use `var` everywhere.

Reply:
Done. I should've used var since this doesn't go through conversion.